package.json

```diff
@@ -155,7 +157,7 @@
     "rollup-plugin-node-resolve": "^4.2.3",
     "rollup-plugin-postcss": "^2.0.3",
     "rollup-plugin-svgo": "^1.0.3",
-    "rollup-plugin-terser": "^4.0.4"
+    "rollup-plugin-terser": "^5.2.0"
```

Comment:
Why did you need to update this?

Reply:
No specific reason. I saw a warning in items to ask for updating.